cli/deploy-profiling-job.sh

```diff
@@ -0,0 +1,137 @@
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
+## Please reach out to the Azure ML docs & samples team before before editing for the first time.
+
+## Preparation Steps:
+## 1. az upgrade -y
+## 2. az extension remove -n ml
+## 3. az extension remove -n azure-cli-ml
+## 4. az extension add -n ml
+## 5. az login
+## 6. az account set --subscription "<YOUR_SUBSCRIPTION>"
+## 7. az configure --defaults group=<RESOURCE_GROUP> workspace=<WORKSPACE_NAME>
+
+set -e
+
+# <set_variables>
+export ENDPOINT_NAME="<ENDPOINT_NAME>"
+export DEPLOYMENT_NAME="<DEPLOYMENT_NAME>"
+export PROFILING_TOOL="<PROFILING_TOOL>"
+export COMPUTE_NAME="<COMPUTE_NAME>"
+export COMPUTE_SIZE="<COMPUTE_SIZE>" # required only when compute does not exist already
+export DURATION="" # time for running the profiling tool, default value is 300s
+export CONNECTIONS="" # for wrk and wrk2 only, no. of connections for the profiling tool, default value is set to be the same as the no. of workers, or 1 if no. of workers is not set
+export THREAD="" # for wrk and wrk2 only, no. of threads allocated for the profiling tool, default value is 1
+export TARGET_RPS="" # for labench and wrk2 only, target rps for the profiling tool, default value is 50
+export CLIENTS="" # for labench only, no. of clients for the profiling tool, default value is set to be the same as the no. of workers, or 1 if no. of workers is not set
+export TIMEOUT="" # for labench only, timeout for each request, default value is 10s
+# </set_variables>
+
+export ENDPOINT_NAME=endpt-`echo $RANDOM`
+export DEPLOYMENT_NAME=blue
+export PROFILING_TOOL=wrk
+export COMPUTE_NAME=profilingTest
+export COMPUTE_SIZE=Standard_F4s_v2
+
+# <create_compute_cluster_for_hosting_the_profiler>
+az ml compute show --name $COMPUTE_NAME
+if [[ $? -ne 0 ]]; then
+  echo "Creating Compute $COMPUTE_NAME ..."
+  az ml compute create --name $COMPUTE_NAME --size $COMPUTE_SIZE --identity-type SystemAssigned --type amlcompute
```

Comment:
no need to check compute existence. if run `compute create` with an existing name, it will auto switch to `update` operation. please have a try to confirm tho.